lib/worker.js

```diff
@@ -42,7 +42,8 @@ Communication.on('JOB', function(job) {
     configFile = params.configFile
   }
 
-  let modulesPath = find(params.fileDir, 'node_modules')
+  const modulesPath = find(params.fileDir, 'node_modules')
+  const eslintignoreDir = Path.dirname(find(params.fileDir, '.eslintignore'))
```

Comment:
No, they are often not in the same dir. .eslintrc files cascade and are merged, while eslintignore files must be in the project root. See http://eslint.org/docs/user-guide/configuring.html#configuration-cascading-and-hierarchy for more details. 

Besides, the goal here is to provide file paths that can be matched against the patterns in the eslintignore, so it makes sense to do it this way I think.